src/libhirte/bus/utils.c

```diff
@@ -80,11 +80,31 @@ UnitInfo *new_unit() {
         }
 
         unit->ref_count = 1;
+        unit->is_from_bus_msg = false;
         LIST_INIT(units, unit);
 
         return steal_pointer(&unit);
 }
 
+int copy_unit(UNUSED UnitInfo *dest, UNUSED UnitInfo *src) {
```

Comment:
```suggestion
int unit_copy(UnitInfo *dest, UnitInfo *src) {
```
Like for `unit_ref`, I think the naming convention is to put the name of the struct type first and the operation second. Would be great to change that for `new_unit` as well.

Reply:
Five seconds after I have issued this PR I understood that I have done this whole copy thing wrong. Will rework it